tests/utils/model_zoo_tester.py

```diff
@@ -56,6 +58,7 @@ def __init__(self, backend, zoo_models, parent_module=None):
             self._add_model_test(test_case, 'Zoo')
 
     @staticmethod
+    @retry
```

Comment:
As described [here](https://pypi.org/project/retrying/) this will keep retrying forever. Can we set a limit on retries? For example:
`@retry(stop_max_attempt_number=10)`,